5ParallelTasks/Program.cs

```diff
@@ -6,6 +6,8 @@ namespace _5ParallelTasks
 {
     internal class Program
     {
+        private const int MaxCount = 5;

```

Comment:
Consider adding a comment explaining what `MaxCount` represents in the context of your application. This can help other developers understand its purpose more quickly. [medium]